report_pre_thesis/src/sections/experiments/other_models.tex

```diff
@@ -0,0 +1,56 @@
+\subsection{Experiment: Other Models}\label{sec:experiment_other_models}
+\citet{cleggRecalibrationMarsScience2017} have only compared their new approach with the original method presented by \citet{wiensPreFlight3}, and have not conducted experiments using alternative methods to establish the superiority of their chosen approach.
+Therefore, we decided to compare the performance of the PLS1-SM and ICA models to other models.
+The objective is to evaluate two distinct scenarios.
+In the first scenario, we aim to conduct a direct comparison between the MOC model and an alternative model. The second scenario revolves around substituting either PLS or ICA with a different model and then calculating a weighted average.
+We have decided to conduct the experiments using the following models:
+
+\begin{itemize}
+	\item \textbf{XGBoost}, a gradient boosting algorithm \cite{chen_xgboost_2016}
+	\item \textbf{ANN}, a neural network model
+\end{itemize}
+
+Building on the findings of \citet{andersonPostlandingMajorElement2022}, who demonstrated Gradient Boosting Regression (GBR) models' superior predictive accuracy in analyzing major oxide compositions in geological samples, we adopt XGBoost for its refined GBR implementation.
+This decision is informed by XGBoost's documented success in enhancing model precision, as detailed in Section~\ref{sec:related_works}.
+
```

Comment:
We should definitely include which parameters etc. that we used for XGboost (if there are any) so that the experiment is replicable.

Reply:
Added to appendix